lib/global_registry/base.rb

```diff
@@ -23,6 +24,24 @@ def self.get(params = {}, headers = {})
     def get(params = {}, headers = {})
       request(:get, params, nil, headers)
     end
+    def self.get_all_pages(params = {}, headers = {})
```

Comment:
Yeah, they all extend and get all of their functionality from `Base`, you define what sub-class  currently only for defining the gr endpoint.